src/OgreVideoManager.cpp

```diff
@@ -105,16 +111,27 @@ namespace Ogre
 		TheoraVideoClip* clip=createVideoClip(new OgreTheoraDataStream(mInputFileName,group_name),TH_RGBA,0,1);
 		int w=nextPow2(clip->getWidth()),h=nextPow2(clip->getHeight());
 
-		TexturePtr t = TextureManager::getSingleton().createManual(name,group_name,TEX_TYPE_2D,w,h,1,0,PF_BYTE_RGBA,TU_DYNAMIC_WRITE_ONLY);
+		TexturePtr t = TextureManager::getSingleton().createManual(name,group_name,TEX_TYPE_2D,w,h,1,0,PF_X8R8G8B8,TU_DYNAMIC_WRITE_ONLY);
```

Comment:
please revert this unrelated change

Reply:
OK, but in next line should still compare `getFormat()` with `PF_X8R8G8B8`?

Reply:
no, it should be `PF_BYTE_RGBA` as well.